utils/checks/snapshots/features.yaml

```diff
@@ -1,3 +1,12 @@
+- name: 0.39.1
```

Comment:
Wrong value for this field

Reply:
https://github.com/humanode-network/humanode/pull/1070/commits/553e92ab57fe28fa730fb6f321bcc39cba924449